### PR TITLE
Promises config event

### PR DESCRIPTION
**Pull request by mattcollier**
@dlongley I want to get this in, (without any op shutdown) before any more merge conflicts arise.

I think the op cutoff can be seen as an additional feature.

### CODE REVIEW

**Comment by mattcollier** [comment]
Also, this the branch that is running on live network right now.

File: lib/ledgerConfiguration.js [hunk 1]
```diff
@@ -28,6 +29,12 @@ api.change = callbackify(async (
     type: 'WebLedgerConfigurationEvent',
     ledgerConfiguration
   };
+  if(!genesis) {
+    const creatorId = (await _voters.get({ledgerNodeId: ledgerNode.id})).id;
```
Comment (dlongley) [inline lib/ledgerConfiguration.js]: Could do: `const {id: creatorId} = ` to get rid of parentheticals

**Comment by dlongley** [comment]
@mattcollier, please merge this *and* #101 when you're done.

**Comment by mattcollier** [comment]
merged.

**Comment by dlongley** [comment]
thanks!

**Pull request closed without merging**